root/tree/dataframe/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ endif()
 
 add_subdirectory(LoopSUSYFrame)
 
-if(rootbench-datafiles AND ((CMAKE_CXX_FLAGS STRLESS_EQUAL "-std=c++17") OR (CMAKE_CXX_STANDARD GREATER 14)))
+if(rootbench-datafiles AND ((CMAKE_CXX_FLAGS MATCHES "-std=c\\+\\+17") OR (CMAKE_CXX_STANDARD GREATER 14)))
```

Comment:
Why are both condition needed?  If the regex match is needed, it will need to be update every time there is a new C++ standard release (too often to be ignored, too rare for us to have any change to remember to do it).

Reply:
this is a stopgap solution until we have #156 . In particular this commit fixes a typo in a previous commit that's already in master